README.md

```diff
@@ -1,5 +1,105 @@
-This repo cntains all the resource files for setting up Kata with SRIOV device plugin.
+# kata-sriov-dp
 
-It contains a daemonset to copy binaries for multus,sriov,cni-shim and patched kubelet
-to the host and start the sriov device plugin.
-Also included are multus CRD and a test pod yaml for Kata.
+This repository contains all the resource files for setting up [SRIOV](https://github.com/intel/sriov-network-device-plugin) device plugin on a Kubernetes cluster.
+
+`kata-sriov-dp` provides a Dockerfile that builds [Multus CNI](https://github.com/intel/multus-cni) plugin, [SRIOV CNI](https://github.com/intel-corp/sriov-cni) plugin,
+[SRIOV Device Plugin](https://github.com/intel/sriov-network-device-plugin) and a patched version of kubelet 1.10 that is required for the device plugin.
+A deamonset is also provided that runs the SRIOV device plugin itself.
+
+The steps to set up the device plugin assume you have a Kubernetes cluster setup, with nodes having a SRIOV enabled network card.
+
+## Kata Containers setup
+
+Below instructions to verify the SRIOV Device plugin work with runc.
+To test out Kata with device plugin, setup Kata Containers with Kubernetes. Refer [kata-deploy](https://github.com/egernst/kata-deploy).
+
+## SRIOV Device Plugin Setup
+
+1. Build Docker image.
```

Comment:
We can skip the docker build/push steps. Minimum Docker-CE version 17.05 or image builders using `BuildKit` to handle multi-stage builds.

Reply:
ok, I can just state here build image and push. I want to mention here about pushing the image ..remember we ran into issues because of that.

Reply:
Yes. Because CRI-O doesn't see Docker images on the host? Also for daemonset that runs on every node in the cluster, the expectation is to have the custom image pulled from a central registry? Either way is fine.